src/validator.js

```diff
@@ -287,27 +344,25 @@ function validateAttributeString(attrStr, options, regxAttrName) {
   const attrNames = {};
 
   for (let i = 0; i < matches.length; i++) {
-    //console.log(matches[i]);
-
     if (matches[i][1].length === 0) {
       //nospace before attribute name: a="sd"b="saf"
-      return {err: {code: 'InvalidAttr', msg: 'attribute ' + matches[i][2] + ' has no space in starting.'}};
+      return {err: {code: 'InvalidAttr', msg: 'attribute ' + matches[i][2] + ' has no space in starting.', position: getPositionFromMatch(attrStr, matches[i][0])}};
```

Comment:
Though it is existing, can you please surround the attribute name in quotes to make it more readable on CLI?

```js
msg: `attribute '${matches[i][2]}' has no space in starting.`
```

Reply:
Hi Amit,

I do agree that this should be fixed, but I also don't want to mix in to mix too many unrelated changes into one PR. 
This is probably worth another issue.

Best regards
Norbert

Reply:
agree. Will do it before release.